lib/ddg-atb.js

```diff
@@ -16,7 +16,8 @@
 
 "use strict";
 
-const {Cu} = require("chrome");
+const {Cu, CC} = require("chrome");
+const XMLHttpRequest = CC("@mozilla.org/xmlextras/xmlhttprequest;1");
```

Comment:
There's a deprecation warning at the top of that link.  What's the advantage to switching until the API is stable?